mathics/autoload/formats/SVG/Export.m

```diff
@@ -4,13 +4,15 @@
 
 
 SVGExport[filename_, expr_, opts___] := 
-  Module[{strm, data}, 
+  Module[{strm, data, p, expr2}, 
     strm = OpenWrite[filename];
     If[strm === $Failed, Return[$Failed]];
-    If[System`$UseSansSerif,
-	    data = StringTake[ToString[MathMLForm[expr]],{23,-8}],
-	    data = StringTake[ToString[MathMLForm[expr]],{23,-8}]];
-    WriteString[strm, "<svg>" <> data <> "</svg>"];
+    expr2 = If[Head[expr]=!=System`Graphics, System`Graphics[{System`Inset[ToString[expr]]}], expr];
```

Comment:
I change this to fix the Export to SVG functionality, in a way that is more independent on how we format the return string in boxes_to_xml